assets/src/blocks/Covers/CoversEditorScript.js

```diff
@@ -0,0 +1,3 @@
+import {registerCoversBlock} from './CoversBlock';
+
+registerCoversBlock();
```

Comment:
Can't we just move the code of the `registerCoversBlock` function to this file?

Reply:
Forget this comment, it does not make sense to introduce changes to a block that will be migrated and removed soon.